package.json

```diff
@@ -20,7 +24,7 @@
   "devDependencies": {
     "@types/google-protobuf": "^3.15.5",
     "@types/jest": "^27.0.2",
-    "@types/node": "^16.11.1",
+    "@types/node": "14.18.3",
```

Comment:
Didn't you upgrade a bunch of other stuff from 14 to 16? Why is this one getting downgraded from 16 to 14?

Reply:
I encountered the issue of not being able to compile JS code to TS code.
And one of the solutions I found was setting the version to `14.18.3` https://github.com/grpc/grpc-node/issues/2002#issuecomment-1003744318

Reply:
Hmm having read this issue it looks like the problem was maybe fixed in a newer version of the library - did you try just updating to the fixed version?

Reply:
Yeah, I also tried the latest version, but I think different errors came up so I tried the fixed version `14.18.3` and it compiled without any issues...

Reply:
Bummer. Oh well for now then